tutorials/cp-cf-processvisibility-config-leadtocash/cp-cf-processvisibility-config-leadtocash.md

```diff
@@ -47,105 +46,105 @@ In this tutorial, you can learn how to gain visibility on a sample lead-to-cash
 
-In this step, you will import the `LeadtoCash.zip` file to the Configure Business Scenarios application.
+In this step, you will import the `LeadtoCash.zip` file to the Configure Visibility Scenarios application.
 
-1. Log on to the **Configure Business Scenarios** tile on SAP Fiori launchpad.
+1. Log onto the **Configure Visibility Scenarios** tile using the Workflow Management home screen.
 
-    ![config scenario](config_flp.png)
+    !![config scenario](config_flp.png)
 
 2. Choose the **Import Business Scenario** icon.
 
-    ![import scenario](screen8-importbusinessscaenrio.png)
+    !![import scenario](screen8-importbusinessscaenrio.png)
 
 3. In the **Import Business Scenario** dialog, browse for the `LeadtoCash.zip` file from your local system. Then choose **Import**.
 
-    ![browse](LeadToCash-import.png)
+    !![browse](LeadToCash-import.png)
 
     You can now see the imported business scenario as shown.
 
-    ![imported scenario](LeadToCash.png)
+    !![imported scenario](LeadToCash.png)
 
 4. Choose the **Lead to Cash** business scenario.
 
-    ![imported scenario](LeadToCash.png)
+    !![imported scenario](LeadToCash.png)
 
       Choose **Activate**.
 
-      ![Activate](Activate.png)
+      !![Activate](Activate.png)
 
-    > You will see a message upon successful activation of the business scenario.
+    > You will see a message upon successful activation of the scenario.
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 4: ](Process the data)]
 
-In this step, you can process the data of the activated business scenarios using the Monitor Scenarios application.
+In this step, you can process the data of the activated business scenarios using the Monitor Visibility Scenarios application.
 
-1. Log on to the **Monitor Scenarios** tile on SAP Fiori launchpad.
+1. Click on the **Monitor Visibility Scenarios** tile by navigating to the home screen.
 
-      ![monitor scenarios](monitor-flp.png)
+      !![monitor scenarios](monitor-flp.png)
 
 2. Choose the **Lead to Cash** scenario definition, then choose **Process Data**.
 
-      ![Process Data](LTC_MonitorScenarios.png)
+      !![Process Data](LTC_MonitorScenarios.png)
 
 3. On successful processing of data, choose the refresh icon to view the processing information listed out under **Processing Information**.
 
-      ![Refresh](ProcessInformation.png)
+      !![Refresh](ProcessInformation.png)
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 5: ](Track business scenario)]
 
-In this step, you will be able to track the progress of the sample business scenario using the visual representation of the Process Workspace application.
+In this step, you will be able to track the progress of the sample scenario using the visual representation of the Process Workspace application.
 
-1. Log on to the **Process Workspace** tile on SAP Fiori launchpad.
+1. Click on the **Process Workspace** tile by navigating to the home screen.
```

Comment:
We don't write "click on the [thing to be clicked]". Just write "click the [thing to be clicked]".